spec/core_functions/meta/content_exists.hrx

```diff
@@ -16,11 +21,6 @@ b {
   c: false;
 }
 
-<===> false/top_level/options.yml
```

Comment:
This change was caused by the interactive mode reordering the files to put `options.yml` first